core/src/test/java/io/github/tcdl/msb/impl/RequesterImplTest.java

```diff
@@ -81,9 +77,20 @@ public void testPublishWaitForResponses() throws Exception {
     }
 
     @Test
+    public void testPublishWaitForResponsesAck() throws Exception {
+        RequesterImpl<RestPayload> requester = initRequesterForResponsesWith(1, 1000, 800, arg ->  fail());
```

Comment:
It seems like this unit test doesn't cover the original requirement: wait for acks even if waitForResponses is 0. Can we have such test here?